src/app/core/components/add-marios-form/add-marios-form.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, Input } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MariosService } from '../../services/marios.service';
+import { Router } from '@angular/router';
+
+import { Employee } from '../../models/employee.model';
+
+@Component({
+  selector: 'app-add-marios-form',
+  templateUrl: './add-marios-form.component.html',
+  styleUrls: ['./add-marios-form.component.scss'],
+})
+export class AddMariosFormComponent {
+  @Input() public maxTitleLength = 128;
+  @Input() public maxMessageLength = 255;
+
+  form: FormGroup = new FormGroup({
+    receiversId: this.createFormControlWithValidation([], []),
+    reaction: this.createFormControlWithValidation('', []),
+    title: this.createFormControlWithValidation('', [
+      Validators.required,
+      Validators.minLength(1),
+      Validators.maxLength(this.maxTitleLength),
+    ]),
+    message: this.createFormControlWithValidation('', [
+      Validators.required,
+      Validators.minLength(1),
+      Validators.maxLength(this.maxMessageLength),
+    ]),
+  });
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private mariosService: MariosService,
+    private router: Router
+  ) {}
+
+  private createFormControlWithValidation(initialValue: any, validators: any) {
```

Comment:
You still have this function, do you need this?

Reply:
Sure i dont need this and i change it like this:
```typescript
form: FormGroup = new FormGroup({
    receiversId: this.formBuilder.control([], []),
    reaction: this.formBuilder.control('', []),
    title: this.formBuilder.control('', [
      Validators.required,
      Validators.minLength(1),
      Validators.maxLength(this.maxTitleLength),
    ]),
    message: this.formBuilder.control('', [
      Validators.required,
      Validators.minLength(1),
      Validators.maxLength(this.maxMessageLength),
    ]),
  });
```
ad4e8ed421ae1075c544ed116e2c9e99e125bafe